sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultTaskSpec.java

```diff
@@ -21,34 +21,42 @@
     @NotNull
     @Size(min = 1)
     private String name;
+
     @NotNull
     private GoalState goalState;
+
     @Valid
     private CommandSpec commandSpec;
+
     @Valid
     private HealthCheckSpec healthCheckSpec;
 
-    private Collection<URI> uris;
+    @Valid
+    private ReadinessCheckSpec readinessCheckSpec;
 
-    private Collection<ConfigFileSpecification> configFiles;
     @Valid
     @NotNull
     private ResourceSet resourceSet;
 
+    private Collection<URI> uris;
+    private Collection<ConfigFileSpecification> configFiles;
+
     @JsonCreator
     public DefaultTaskSpec(
             @JsonProperty("name") String name,
             @JsonProperty("goal") GoalState goalState,
             @JsonProperty("resource_set") ResourceSet resourceSet,
             @JsonProperty("command_spec") CommandSpec commandSpec,
             @JsonProperty("health_check_spec") HealthCheckSpec healthCheckSpec,
+            @JsonProperty("readiness_check_spec") ReadinessCheckSpec readinessCheckSpec,
```

Comment:
Maybe not in scope for this, but I think we unofficially settled on kebab-case for all field names in specs, rather than snake_case.